data_checker/main.resource_policy_check.py

```diff
@@ -0,0 +1,134 @@
+import os
+import logging
+import requests
+import psycopg2
+
+import const
+
+from data_pump.utils import create_dict_from_json, read_json, \
+    convert_response_to_json, do_api_get_one
+from support.dspace_proxy import rest_proxy
+
+logging.basicConfig(level=logging.INFO)
+_logger = logging.getLogger("root")

```

Comment:
also, this should be ("resourcepolicy_checker") or something... now logs look like.. this:
![image](https://github.com/dataquest-dev/dspace-python-api/assets/88670521/8d7e27fb-ca00-4241-a060-7e304d7ef308)
Instead of some meaningful message that the specific log speaks about resource policy check.

Reply:
so do you want only to set logger name from `root` to `resource_checker`?